src/main/java/FriendAdder.java

```diff
@@ -0,0 +1,42 @@
+import java.util.ArrayList;
+
+public class FriendAdder {
+    /** Adds users to friend list by sending invitations, then accepting invitations.
```

Comment:
ignore this if you've seen it done differently in example code we've been given, but javadocs for classes should go outside the class, no?